Unity/Champloo/Assets/Scripts/Weapons/Projectiles/Projectile.cs

```diff
@@ -32,6 +34,11 @@ protected virtual void Start()
         {
             OnTriggerEnter2D(col);
         }
+        player = FindPlayer();
+        if (player == null)
+        {
+            Debug.LogError("Could not find player!");
```

Comment:
Yes; I could switch it over to throwing a null pointer exception to make it more explicit